### PR TITLE
Twm 26 contact us page error

### PR DESCRIPTION
**Pull request by cdoyle-temple**
I've added mailform and simple_form. 
Also added methods for the form submission, Contact Us.
This form is actually of the Page type, but I call the form submission routine
Thinking other forms, also of type Page, can use similar methods
Need env variable FORMS_EMAIL_PASSWORD set on server to test from staging (local does not send out emails but the server console shows the mails being generated ant sent)

### CODE REVIEW

File: app/views/layouts/mailer.html.erb [hunk 1]
```diff
@@ -8,6 +8,6 @@
   </head>
 
   <body>
-    <%= yield %>
+    yield: <%= yield %>
```
Comment (bibliotechy) [inline app/views/layouts/mailer.html.erb]: do you want this to say yield in the text?

File: db/migrate/20190226151003_create_forms.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class CreateForms < ActiveRecord::Migration[5.0]
```
Comment (bibliotechy) [inline db/migrate/20190226151003_create_forms.rb]: What does this do? If it is only adding timestamps, I can't see how it would be useful.

File: db/schema.rb [hunk 3]
```diff
@@ -333,7 +338,7 @@
     t.string   "pdf"
   end
 
-  create_table "tests", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci" do |t|
```
Comment (bibliotechy) [inline db/schema.rb]: why is this change here?

File: app/mailers/application_mailer.rb [hunk 4]
```diff
@@ -1,4 +1,21 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default to: 'cdoyle@temple.edu'
   layout 'mailer'
+
+  def contact(form)
+    @user = form
+    mail(body: '', from: @user.email, subject: @user.subject, comments: @user.comments,
+    	  address1: @user.address1,
+			  address2: @user.address2,
```
Comment (bibliotechy) [inline app/mailers/application_mailer.rb]: Please make this indentation more consistent.

File: app/models/form.rb [hunk 5]
```diff
@@ -0,0 +1,32 @@
+class Form < MailForm::Base
+	# include MailForm::Delivery
+
+	attributes :name,      :validate => true
+  attributes :email,     :validate => /\A([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})\z/i
+  attributes :comments
+  attributes :subject,
```
Comment (bibliotechy) [inline app/models/form.rb:7]: You need to add some indentation to these for readbility

File: app/models/page.rb [hunk 6]
```diff
@@ -1,7 +1,6 @@
 class Page < ApplicationRecord
 	require 'carrierwave/orm/activerecord'
-	mount_uploader :file, DocumentUploader
```
Comment (bibliotechy) [inline app/models/page.rb]: This is related to the contact page?

**Review by bibliotechy — CHANGES_REQUESTED** [review]
Comments in line

File: app/views/layouts/mailer.html.erb (hunk 1 above)
Comment (cdoyle-temple) [reply app/views/layouts/mailer.html.erb]: Saw this too after requesting review. I do not.

Comment (cdoyle-temple) [reply app/views/layouts/mailer.html.erb]: It never shows anywhere though. But without the file, the app errors on submission.

File: db/migrate/20190226151003_create_forms.rb (hunk 2 above)
Comment (cdoyle-temple) [reply db/migrate/20190226151003_create_forms.rb]: It would not be useful. Cleaned up.

File: db/schema.rb (hunk 3 above)
Comment (cdoyle-temple) [reply db/schema.rb]: I do not know. Forms table is not needed. False path no longer followed. Will revert this file as there should be no schema changes.

**Review by cdoyle-temple — COMMENTED** [review]
The indentation did not appear the same locally for the application mailer. This should now be addressed however.

And the inadvertently removed uploader is restored.

**Review by bibliotechy — APPROVED** [review]

**Comment by cdoyle-temple** [comment]
I've added the two other forms and now need to test the email sending on the staging server. The FORMS_EMAIL_PASSWORD needs to be set on staging and production. It is the same as manifold's.
